data/remnant/remnant jobs.txt

```diff
@@ -179,7 +179,7 @@ mission "Remnant: Bounty 4"
 			names "korath"
 			cargo 3
 			variant
-				"Korath Dredger (Hyperdrive)"
+				"Korath Dredger"
```

Comment:
We're avoiding touching anything with @Zitchas 's missions, so you should run this by him.

Reply:
I'm not particularly concerned at this point.

A) it is already trivial to farm Jump Drives.

B) I am already planning on doing a full review of the relative combat difficulty of the Remnant campaign as a whole once this PR gets in and we have greater flexibility for making a more diverse range of opponents.

Reply:
The (Hyperdrive) variant *had* a JD, it just had a hyperdrive, too.  Basically it was the Dredger's equivalent of an (Ember) variant.  Since all Dredgers, by virtue of their hyperdrives and cavernous outfit space for cooling and generation are Ember-capable anyway, I found this variant to be completely and utterly redundant.